src/ports/nfts/component.ts

```diff
@@ -47,6 +51,20 @@ export function createNFTComponent<T extends { id: string }>(options: {
     }
   }
 
+  async function getBannedNames() {
+    try {
+      const bannedNames = await nodeFetch(`${listsServer}/banned-names`, {
+        method: 'POST',
+      })
```

Comment:
Do you think this could be part of a new port?

Reply:
I think that if there's a new requirement that needs to re-use this piece of code, it would be worth it to create a new port but for the time being, I'd let it here.